examples/llm_inference/android/.gitignore

```diff
@@ -0,0 +1,15 @@
+*.iml
```

Comment:
Can you change this to a non-Draft PR and rename to "LLM Inference Example App" or something?